app/templates/brief.html

```diff
@@ -130,9 +130,8 @@
   <div class="grid-row">
     <div class="column-one-third">
         {% with
-           url = "/suppliers/opportunities/{}/responses/result".format(brief.id),
-           text = "View your application",
-           bigger = True
```

Comment:
No bigger?

Reply:
Nope! It looks awful. Confirmed with Laurence that the non-big link is better.

Reply:
Cool, just checking :)